### ISSUE TITLE
argparse.ArgumentParser does not raise on duplicated subparsers, even though it does on duplicated flags

### ISSUE THREAD

**Issue by 379dc349-3a10-424f-b9d2-a0104f092359** [python/cpython#83897]
BPO | [39716](https://bugs.python.org/issue39716)
--- | :---
Nosy | @rhettinger, @anntzer
PRs | <li>python/cpython#18605</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2020-02-21.18:20:54.496>
labels = ['library', '3.9']
title = 'argparse.ArgumentParser does not raise on duplicated subparsers, even though it does on duplicated flags'
updated_at = <Date 2020-02-22.11:04:34.546>
user = 'https://github.com/anntzer'
```

bugs.python.org fields:
```python
activity = <Date 2020-02-22.11:04:34.546>
actor = 'Antony.Lee'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2020-02-21.18:20:54.496>
creator = 'Antony.Lee'
dependencies = []
files = []
hgrepos = []
issue_num = 39716
keywords = ['patch']
message_count = 3.0
messages = ['362421', '362428', '362458']
nosy_count = 3.0
nosy_names = ['rhettinger', 'paul.j3', 'Antony.Lee']
pr_nums = ['18605']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue39716'
versions = ['Python 3.9']
```

</p></details>

**Comment by ppolewicz** [comment]
Please note that this issue is labeled as `3.9` and fixed for it was submitted in February 2020, but the fix was actually released with `3.11`

**Comment by terryjreedy** [comment]
@ppolewicz PRs are nearly always merged into main and only sometimes backported.  Tracker version labels indicate the version(s) reported for, not necessarily the versions patched.  Since we moved the repository to github, PR titles indicate the version(s) actually patched (with 'main' the default).  The tracker move to github messed up merge reporting for old PRs.  I repointed the PR for this issue to the gh number so it is listed immediately above.
